site/get-started.markdown

```diff
@@ -0,0 +1,143 @@
+
+It is simple to use, works the same way on all platforms (Linux, macOS, Windows), and gives you one central place/method to take care of your Haskell development setup.
+
+Follow instructions at [GHCup webpage](https://www.haskell.org/ghcup/#) to install GHCup. Then, use it to install the Haskell Toolchain, which consists of:
+
+1. **GHC** -> Haskell compiler. We will use it below to run our examples, but in practice, you will mostly be using a build tool like `cabal` or `Stack` to build your code, instead of `GHC` directly.
+2. **HLS** -> Haskell Language Server -> You won't use this directly, instead your code editor will use it in the background to provide you with a great experience while editing Haskell code.
+3. **cabal** -> Haskell build tool -> You will use this to structure your Haskell projects, build them, run them, define dependencies, ...
+4. **Stack** -> Haskell build tool -> alternative to `cabal`
+
+
+<div class="bs-callout bs-callout-info">
+  <p>
+    <h4>cabal and Stack -> which one should I install?</h4>
+    cabal is the original build tool, while Stack was created as an alternative to cabal a time ago to solve some of the problems that cabal had and provide a more user-friendly experience. In the meantime, cabal solved most of those issues (including "cabal hell") and caught up with Stack regarding user experience, so the main difference between them at the moment is how they resolve dependencies, which for you as a beginner isn't a concern. Therefore, both are a good choice, and if not sure, you can install both and then use whatever the learning resources you will later use will point you to.
+  </p>
+</div>
+
+To confirm it is all installed correctly, you can run the following commands:
+```
+> ghc --version
+> haskell-language-server-wrapper --version
+> cabal --version                               # If you installed cabal
+> stack --version                               # If you installed Stack
+```
+
+### Editor
+HLS (Haskell Language Server), which you just installed via GHCup, is the one that brings all the cool IDE features to editors, so as long as your editor has a decent Haskell language extension that utilizes HLS, you are good.
+
+The most straightforward option is to go with **Visual Studio Code** - just install [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell) and you are all set, it should work out of the box and use your installation of HLS.
+
+Of other editors that have good Haskell extensions, the most popular ones are Vim and Emacs.
+
+## Running first lines of code
+
+We have everything set up, let's use it!
+
+`GHC` brings an interactive interpreter called `GHCi` together with it, which is great for playing with Haskell and trying things out, so let's give it a spin.
+So even though Haskell is a compiled language, it comes with an interpreter too -> how cool is that?
```

Comment:
compiled is not a property of a programming language but rather of an implementation, so it's not really correct to call Haskell a compiled language. Instead you could say that GHC can compile a Haskell program using the `ghc` command, or interpret it using the `runghc` command. GHC also provides an REPL-style interactive interpreter via the `ghci` command.

Reply:
Removed